test/19.js

```diff
@@ -0,0 +1,23 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/19').solution;
+//solution = require('../yourSolution').solution;
+
+describe('Array pairs that sum to x', () => {
+    it('should return an array of pairs of integers whose sum equal to x', () => {
+        const result = solution([1,2,3,4,5,6], 6);
+        expect(result).to.deep.equal([[1,5], [2,4]]);
+
+        const result2 = solution([2,1,3,4,5,6], 6);
+        expect(result).to.deep.equal([[1,5], [2,4]]);
+    });
+
+    it('should only use an integer once in its output', () => {
+        const result = solution([1,3,3,3], 4);
+        expect(result).to.deep.equal([[1,3]]);
+    });
+
+    it('should return null of no pairs are found', () => {
+        const result = solution([1,2,3,4,5,6], 40);
+        expect(result).to.equal(null);
```

Comment:
yeah I was going to point that something was off. We can't assume its a sorted array. Real life scenario: We get a list of test scores and we want to pair people up so that the teams are even. We can't assume the list of test scores are sorted.